tests/code_update.py

```diff
@@ -364,6 +365,40 @@ def test_proposal_invalidation(network, args):
     return network
 
 
+@reqs.description(
+    "Test deploying secondary ACIs which will be used to test SNP code update"
+)
+@reqs.snp_only()
+def test_snp_secondary_deployment(network, args):
+
+    LOG.info(f"Secondary ACI information expected at: {args.snp_secondary_acis_path}")
+
+    timeout = 60 * 60  # 60 minutes
+    start_time = time.time()
+    end_time = start_time + timeout
+
+    while time.time() < end_time and not os.path.exists(args.snp_secondary_acis_path):
+        LOG.info(
+            f"({time.time() - start_time}) Waiting for SNP secondary IP addresses file at: ({args.snp_secondary_acis_path}) to be created"
+        )
+        time.sleep(60)
```

Comment:
This seems like a long sleep?

Reply:
Yeah it is, I'll drop it down

Reply:
https://github.com/microsoft/CCF/pull/4861/commits/fa316f3aae44880d20757e9119aaa0283ea6c709